plugins/woocommerce/package.json

```diff
@@ -786,8 +786,10 @@
 				"node_modules/@woocommerce/e2e-core-tests/CHANGELOG.md",
 				"node_modules/@woocommerce/api/dist/",
 				"node_modules/@woocommerce/admin-e2e-tests/build",
+				"node_modules/@woocommerce/classic-assets/build",
 				"node_modules/@woocommerce/block-library/build",
 				"node_modules/@woocommerce/block-library/blocks.ini",
+				"node_modules/@woocommerce/admin-library/build",
```

Comment:
Are these changes required? I get the same changes always after pnpm install

Reply:
Ah yeah, I will revert this, thanks 🙇

Reply:
Looks like this change has already been reverted.